spec/models/storer_spec.rb

```diff
@@ -44,7 +44,7 @@
     it 'stores data to karios & redis publish' do
       # model/storer.rb is not using Kairos, but Redis -> Telnet
       # expect(Kairos).to receive(:http_post_to).with("/datapoints", @karios_data)
-      expect_any_instance_of(Storer).to receive(:redis_publish).with(@readings, @ts, true)
+      expect_any_instance_of(Storer).to receive(:redis_publish)
```

Comment:
I don't understand this. Are we not sending @readings to Redis? :thinking:

Reply:
Read the code better:

`redis_publish` as it is now (see below) it doesn't require any params to be passed to the function as it only needs the devide that's already available from the model context

https://github.com/fablabbcn/smartcitizen-api/blob/5dc279e11d30f715bdbe90afecae399bb2e6c6ea/app/models/storer.rb#L36-L42